.vale.ini

```diff
@@ -9,7 +9,13 @@ MinAlertLevel = warning
 # 4. You can test locally by replacing the url with the file path of the generated zip
 Packages = Google, https://github.com/mui/material-ui/raw/HEAD/docs/mui-vale.zip
 
+[formats]
+mdx = md
+
 [*.md]
```

Comment:
This is kind of magic, it lint mdx too, it doesn't need to be: 
```suggestion
[*.{md,mdx}]
```

so I didn't change it 🤷‍♂️